accounts-db/src/account_storage/meta.rs

```diff
@@ -61,23 +61,31 @@ impl<'a: 'b, 'b, U: StorableAccounts<'a>, V: Borrow<AccountHash>>
     }
 
     /// get all account fields at 'index'
-    pub fn get(&self, index: usize) -> (Option<AccountForStorage>, &Pubkey, &AccountHash) {
-        let account = self.accounts.account_default_if_zero_lamport(index);
+    pub fn get<Ret>(
+        &self,
+        index: usize,
+        mut callback: impl FnMut((Option<AccountForStorage>, &Pubkey, &AccountHash)) -> Ret,
```

Comment:
main change is `get` and `account` right below take a callback now instead of returning a ref.

Reply:
@brooksprumo review this with hide whitespace